src/full/Agda/Syntax/Concrete/Pretty.hs

```diff
@@ -212,7 +212,11 @@ instance Pretty Expr where
                     , maybe empty (\ e -> "in" <+> pretty e) me
                     ]
             Paren _ e -> parens $ pretty e
-            IdiomBrackets _ e -> "(|" <+> pretty e <+> "|)"
+            IdiomBrackets _ es ->
+              case es of
+                []   -> "(|)"
+                [e]  -> "(|" <+> pretty e <+> "|)"
+                e:es -> "(|" <+> pretty e <+> fsep (map (("|" <+>) . pretty) es) <+> "|)"
             DoBlock _ ss -> "do" <+> vcat (map pretty ss)
```

Comment:
Note that here you could [special characters](https://github.com/agda/agda/blob/b0229caffc87e99696dff73902cc72ab1a2cd9ec/src/full/Agda/Syntax/Concrete/Pretty.hs#L74) to answer in unicode
unless the users have explicitly asked for `--no-unicode`.

Reply:
Good point, but it is a bit subtle.

* The implementation does not provide a Unicode variant for `(|)`, so it is inconsistent to use Unicode.
* Use `⦇⦈` anyway would suggest that `⦇⦈` is a keyword. 
* However, making `⦇⦈` a keyword is a breaking change.

Reply:
We could add a new builtin called `ALTERNATIVE_EMPTY` in `Agda.Builtin.Alternative` or something like that, and bind it to `⦇⦈` by default. That way users can choose to rename it however they want (or not import the

Reply:
Can you elaborate further? I have the impression that a built-in always has a particular type attached to it. 

`(|)` is a syntactic sugar, and it will expand to `empty` before scope checking. By design, `(|)` can be any type, so if you define

```agda
empty = 0
```
then `(|)` will just be `0`.

Reply:
For the sake of consistency, `⦇⦈` is a keyword now. Although it is a breaking change, I doubt anyone would use `⦇⦈` alone considering that `⦇` and `⦈` have been reserved.  

Then the pretty-printer prints Unicode message if `--no-unicode` is not set and the error message thrown by scope checking also uses Unicode accordingly, i.e.

https://github.com/agda/agda/blob/b00c843209e350cf1b07bffd816f13f19e53e808/src/full/Agda/Syntax/IdiomBrackets.hs#L63-L65

The change shall please @gallais  :-)